test/testdrive/bytes.td

```diff
@@ -42,3 +42,4 @@ name       nullable  type
 --------------------------
 named_col  false     bytea
 mz_offset  false     int8
+nonsense
```

Comment:
Testdrive has some tests that test its CLI output in src/testdrive/tests/cli.rs, which would be a perfect fit for testing this permanently, if you were so inclined!

Reply:
oh, nice!